src/MyHealth/Today.spec.tsx

```diff
@@ -1,192 +0,0 @@
-import React from "react"
```

Comment:
This seems like a pretty big change in its own right. What do you think about separating the prs of 'Removing the save CheckIn feature' And 'Remove the Today Tab' ? its not clear to me that they need to go together? and its preferable to have PRs be as small as possible.

Additionally, Why are we removing the Today vs Over time tabs? I think we still want to have those separate tabs? and we just wanted to remove the 'Feeling Good' button for the time being until we can figure out what its meant to do?